### ISSUE TITLE
DPL-573 Bioscan - Job to create compound sample, mBrave file and export message to Traction

### ISSUE THREAD

**Issue by andrewsparkes** [sanger/sequencescape#3751]
**User story**
As PSD we need to trigger an asynchronous job from the Limber XP tube to create the compound sample, the mBrave output file with 9216 samples using sample metadata data, and generate the message to export the tube to Traction. 

**Who are the primary contacts for this story**
PSD
Bioscan faculty (who TBC)

**Who is the nominated tester for UAT**
Bioscan faculty (who TBC)

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Limber button click on XP tube (see DPL-662) should trigger an async job to be created
- [ ] Job should create:
     - [ ] the compound sample
     - [ ] the mBrave file in network directory with XP tube barcode as the 'source' in the filename
     - [ ] write a RabbitMQ message to a queue to export the tube to Traction
     - [ ] *** Keep the state of the job updated as it progresses on Tube 

**Dependencies**
This story is blocked by the following dependencies:
- Limber [DPL-662](https://github.com/sanger/limber/issues/1237) for button to trigger export

References
This story has a non-blocking relationship with:
- Sequencescape [DPL-663](#3750) Creation of the compound sample
- tol-lab-share [DPL-666](https://github.com/sanger/tol-lab-share/issues/29) Creation of the export message

**Additional context**
Possibly sub-task this story?

The mBrave file would be created in Sequencescape and written to a network drive location with the XP tube barcode as part of the filename. This 'source' barcode would also be visible in Traction to allow the user to associate the file with the sequencing.

**Comment by sdjmchattie** [comment]
Duplicate of #3754
